tests/README.txt

```diff
@@ -39,6 +40,69 @@ CONTENTS
       README.txt          This README file
 
 
+
+RUNNING THE SUITE
+
+  Linux and OS X
+
+    In the top level directory of the distribution, you can execute the
+    following command to find your target printer's DNS-SD Instance Name:
+
+      ./ippfind -s
+
+    If your printer implements the "_print" DNS-SD subtype for the "_ipp._tcp"
+    service type that is required by IPP Everywhere, you can filter the
+    results using this command, if you have many printers:
+
+      ./ippfind "_print._ipp._tcp" -s
+
+    Once you have acquired the target printer's DNS-SD Instance Name, you can
+    execute the following commands to run the 3 sets of tests ("Example Test Printer"
+    is the example DNS-SD Instance Name in the examples below):
+
+      ./runtests.sh bonjour-tests.sh "Example Test Printer"
+      ./runtests.sh ipp-tests.sh "Example Test Printer"
+      ./runtests.sh document-tests.sh "Example Test Printer"
+
+    The corresponding PWG Raster sample files (see link in the introduction) MUST
+    be placed in a subdirectory of the "tests" directory. For example:
```

Comment:
ipptool should look both in the current directory and the tests directory for the sample files.